bower.json

```diff
@@ -1 +1 @@
-@$% Not a Bower package. Use a better package management tool instead. ^&*
\ No newline at end of file
+@$% Not a Bower package. Use a better package management tool instead. ^&*
```

Comment:
Can you skip touching this file altogether?

Reply:
Done